File: Source/Rio.Web/src/app/shared/models/account/account-include-parcels-dto.ts
import { ParcelSimpleDto } from '../parcel/parcel-simple-dto';
import { AccountDto } from './account-dto';
export class AccountIncludeParcelsDto {
    Account: AccountDto;
    Parcels: Array<ParcelSimpleDto>;
    
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Rio.Models/DataTransferObjects/Account/AccountIncludeParcelsDto.cs
using System.Collections.Generic;
using Rio.Models.DataTransferObjects.Parcel;

namespace Rio.Models.DataTransferObjects.Account
{
    public class AccountIncludeParcelsDto
    {
        public AccountDto Account { get; set; }
        public List<ParcelSimpleDto> Parcels { get; set; }
    }
}
